lms/templates/discovery/course_card.underscore

```diff
@@ -3,7 +3,7 @@
         <header class="course-image">
             <div class="cover-image">
                 <img src="<%- image_url %>" alt="<%- content.display_name %> <%- content.number %>" />
-                <div class="learn-more" aria-hidden=true><%- gettext("LEARN MORE") %></div>
+                <div class="learn-more" aria-hidden="true"><%- gettext("LEARN MORE") %></div>
```

Comment:
@clrux Why should this text be aria-hidden? It doesn't seem right to me, but I can't say I understand the structure of this template. I guess there is an aria-label on the outermost article, but I don't see how the a link states what it goes to.